guides/common/modules/proc_upgrading-hosts-to-next-major-rhel-release.adoc

```diff
@@ -1,17 +1,31 @@
-[id="Upgrading_Hosts_From_RHEL7_to_RHEL8_{context}"]
-= Upgrading Hosts From RHEL 7 to RHEL 8
+[id="Upgrading_Hosts_to_Next_Major RHEL_release_{context}"]
+= Upgrading Hosts to Next Major {RHEL} Release
```

Comment:
```suggestion
= Upgrading Hosts to The Next Major {RHEL} Release
```

I am unsure about the wording here. cc @Levi-Leah

Reply:
> I am unsure about the wording here.

imho the wording is fine, I left a comment on prerequisites tho